src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  position: relative;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  margin: 20px 0;
+  align-items: center;
+  border-radius: 4px;
+}
+
+.search-bar:hover,
+.search-bar:focus {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search-bar--big {
+  height: 70px;
+}
+
+.search-bar--small {
+  height: 42px;
+}
+
+.search-bar--big::after {
+  position: absolute;
+  left: 26px;
+  display: block;
+  background-image: url(images/Search.svg);
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  content: "";
+  width: 19px;
+  height: 19px;
+}
+
+.search-bar--small::after {
+  position: absolute;
+  left: 13px;
+  display: block;
+  background-image: url(images/Search.svg);
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  content: "";
+  width: 11px;
+  height: 11px;
+}
```

Comment:
a lot of repeated styles. Add pseudo-element after to a common class search-bar and then just add difference to 
search-bar--big::after and search-bar--small::after